src/translations/nbTranslations.json

```
@@ -465,7 +465,9 @@
         "claim_category_restriction": "Kategorier som begrensningen skal gjelde for",
         "editing_access": "Tilgang til å redigere",
         "publishing_access": "Tilgang til å publisere",
-        "set_channel_claim": "Sett eierskap"
+        "publisher_claims_overview": "Oversikt over utgivere",
+        "set_channel_claim": "Sett eierskap",
+        "serial_publication_claims_overview": "Oversikt over serie/tidsskrift"
```

mangler noe her? Siden det er en oversikt over kanaleierskap, ikke over utgivere og tidsskrifter/serier. "Utgivereierskap" klinger ikke akkurat bra det heller men 🤔 

```suggestion
        "publisher_claims_overview": "Oversikt over utgivereierskap",
        "set_channel_claim": "Sett eierskap",
        "serial_publication_claims_overview": "Oversikt over serie-/tidsskrifteierskap"
```

---

"Eierskap for utgivere" og "Eierskap for serie/tidsskrift" er kanskje bedre, sånn at det matcher naming som brukes for administrasjon?